### DISCUSSION TITLE
Locking playlist from chaning

### DISCUSSION THREAD

**Discussion by axet** [audacious-media-player/audacious#755] [General]
Hello!

Listening audiobooks with Audacious can be lost current playing position and playlist got reset when new audio file opened with Audacious. It happened because current playing playlist got replaced with new file. Old playlist got lost, no way to restore it.

New playlist "Locked" could be an a solution. Locked playlist will never be replaced / changed by new files opened using "Open With" OS feature. This option can be added to Playlist and Right click on playlist name next to "Play / Rename ... / Remove"

Thanks.

**Comment by jlindgren90** [comment]
1. Make sure you have enabled (the default) Settings -> Playlist -> Open files in a temporary playlist.
2. When you want to make sure a playlist is not overwritten by opening files, simply (re)name it to anything other than "Now Playing" (or whatever translation of that is used in your language) - then it will not be overwritten and a new playlist will be created instead.

**Comment by axet** [comment]
Thanks. It indeed works. Maybe it could be done a bit more obvious.
